src/modules/markets/selectors/tags.js

```diff
@@ -1,46 +1,49 @@
-import memoizerific from 'memoizerific';
+import { createSelector } from 'reselect';
+import store from 'src/store';
+import { selectSelectedTagsState } from 'src/select-state';
+import { selectFilteredMarkets } from '../../markets/selectors/markets-filtered';
```

Comment:
Currently, no, but I should get that setup.  

Right now if you have utilized a path alias, you’ll need to use the shortest path to the referenced file in order for proxyquire to work.